src/ayab/encoders.h

```diff
@@ -63,14 +63,14 @@ constexpr uint8_t END_LEFT[NUM_MACHINES] = {0U, 0U, 0U};
 constexpr uint8_t END_RIGHT[NUM_MACHINES] = {255U, 255U, 140U};
 constexpr uint8_t END_OFFSET[NUM_MACHINES] = {28U, 28U, 14U};
 
-constexpr uint8_t GARTER_SLOP = 14;
+constexpr uint8_t GARTER_SLOP = 2;
 
 constexpr uint8_t START_OFFSET[NUM_MACHINES][NUM_DIRECTIONS][NUM_CARRIAGES] = {
     // KH910
     {
         //   K,  L,  G
-        {40, 40, 8}, // Left
-        {16, 16, 32} // Right
+        {40, 40, 31}, // Left
```

Comment:
Where are these new offsets for G carriage coming from ?
+7 needles from what I see below
31 % 16 = 15 vs 8 % 16 = 8
55 % 16 = 7 vs 32 % 16 = 0

Reply:
They came from manual testing.  I'm going to pull them into a separate PR.

Reply:
I guess that 55 would give the same behavior as 7 as everything is modulo 16 for solenoids ? (meaning "G shifted by 9 vs K/L",, i.e. 40-9 and 16-9)
As this is related to the physical position of the G magnet, central for K/L and shifted for G, I would have said 12 iso 9 for G a priori, i.e 28 and 4, as this is more or less the distance in needles between the magnet position and the center of the carriage.

Reply:
I will keep that in mind.